### PR TITLE
Alpha  Review for issue #1558 via pull request

### PR DESCRIPTION
**Pull request by msrader**
I think I have addressed all of the comments you posed from the last review.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/FormatString.java [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+	private static Logger log = Logger.getLogger(FormatString.class);
+	
+	@Override
+	public int getId() {	
+		return 30002;
+	}
+
+	@Override
+	public String getName() {
+		// Strip off the "Format String Error: " part if implementing a real one ;)
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/FormatString.java]: The comment should be removed ;)

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml [hunk 2]
```diff
@@ -8,6 +8,7 @@
 	<![CDATA[
 	Updated add-on's info URL.<br>
 	Minor code changes.
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml]: This line should be terminated with "<br>", for proper formatting when shown in ZAP.

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html [hunk 3]
```diff
@@ -27,6 +27,10 @@
 <H2>Expression Language Injection</H2>
 The software constructs all or part of an expression language (EL) statement in a Java Server Page (JSP) using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended EL statement before it is executed. In certain versions of Spring 3.0.5 and earlier, there was a vulnerability (CVE-2011-2730) in which Expression Language tags would be evaluated twice, which effectively exposed any application to EL injection. However, even for later versions, this weakness is still possible depending on configuration.
 
++<H2>Format String Error</H2>
+Looks for indicators of format string handling errors in complied code.  It does this by putting out strings of input text based upon characters
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html]: There's still a typo "complied".

**Comment by thc202** [comment]
The changes should be done with just one commit. Could you fixup the other commits?

**Comment by msrader** [comment]
I'm not that familiar with github so I have no easy way to approach it.

**Pull request closed without merging**